packages/tracking/src/lib/Client.php

```diff
@@ -57,8 +56,8 @@ static function record_event( $event ) {
 			return false;
 		}
 		
-		if ( ! $event instanceof Jetpack_Tracks_Event ) {
-			$event = new Jetpack_Tracks_Event( $event );
+		if ( ! $event instanceof Event ) {
```

Comment:
I think `instanceof` works with the fully qualified name of the class, e.g. `Automattic\Jetpack\Tracks\Event`